content/en/spec/activitypub.md

```diff
@@ -852,58 +852,69 @@ Mastodon generates colorful preview thumbnails for attachments. This is implemen
 
 ### Sensitive content {#sensitive}
 
-Mastodon uses the `as:sensitive` extension property to mark certain posts as sensitive. When a post is marked as sensitive, any media attached to it will be hidden by default, and if a `summary` is present, the status `content` will be collapsed behind this summary. In Mastodon, this is known as a **content warning**.
+Mastodon uses the `as:sensitive` extension property to mark certain posts as sensitive. When a post is marked as sensitive, any media attached to it will be hidden by default, and if a `summary` is present, the status `content` will be collapsed behind this summary. In Mastodon, this is known as a **Content Warning** (CW), which allows users to hide potentially sensitive content behind a warning label.
 
 ### Quote posts and quote controls {#Quote}
 
 Mastodon implements experimental support for handling remote quote posts according to [FEP-044f](https://codeberg.org/fediverse/fep/src/branch/main/fep/044f/fep-044f.md). Additionally, it understands `quoteUri`, `quoteUrl` and `_misskey_quote` for compatibility.
 
 Should a post contain multiple quotes, Mastodon only accepts the first one.
 
-Furthermore, Mastodon does not handle the full range of interaction policies, but instead converts the authorized followers to a combination of “public”, “followers” and “unknown”, defaulting to “nobody”.
+Furthermore, Mastodon does not handle the full range of interaction policies, but instead converts the authorised followers to a combination of "public", "followers" and "unknown", defaulting to "nobody".
 
 At this time, Mastodon does not offer authoring quotes, nor does it expose a quote policy, or produce stamps for incoming quote requests.
 
-## Other functionality
-
-### Secure mode {#secure-mode}
+### Follower synchronisation mechanism
```

Comment:
why was this moved out of "other functionality"? it's not part of the data model of federation